dhis-2/dhis-web-api-test/src/test/java/org/hisp/dhis/webapi/controller/TrackedEntityAttributeControllerTest.java

```diff
@@ -166,6 +167,102 @@ void getIndexableOnlyAttributes()
 
     }
 
+    @Test
+    void getIndexableOnlyAttributesWithDateAndTextValueType()
+    {
+
+        teaA.setValueType( ValueType.DATE );
+        manager.update( teaA );
+        teaB.setValueType( ValueType.TEXT );
+        manager.update( teaB );
+        JsonObject json = GET( "/trackedEntityAttributes?indexableOnly=true" ).content( HttpStatus.OK );
```

Comment:
I like newlines around the thing that is tested. So its easy to see what is tested. Some call it AAA arrange, act, assert. So you end up with 3 blocks each with one of these responsibilities. Makes tests easier to read I think.